reference/v8.json

```diff
@@ -157,18 +157,18 @@
     },
     "maxzoom": {
       "type": "number",
-      "default": 14,
+      "default": 18,
       "doc": "Maximum zoom level at which to create vector tiles (higher means greater detail at high zoom levels)."
     },
     "buffer": {
       "type": "number",
-      "default": 64,
-      "doc": "Tile buffer size on each side (higher means fewer rendering artifacts near tile edges but slower performance)."
+      "default": 128,
+      "doc": "Tile buffer size on each side (relative to a 512px tile; higher means fewer rendering artifacts near tile edges but slower performance)."
     },
     "tolerance": {
       "type": "number",
-      "default": 3,
-      "doc": "Douglas-Peucker simplification tolerance (higher means simpler geometries and faster performance)."
+      "default": 0.375,
+      "doc": "Simplification tolerance (higher means simpler geometries and faster performance)."
```

Comment:
I think it's an implementation detail that's not important to most users, but sounds scary.